releases/release-4.0.6.md

```diff
@@ -0,0 +1,156 @@
+    - Fix issue 19371 - PREPARE statement FROM @Var (where Var contains uppercase characters fail) [#19378](https://github.com/pingcap/tidb/pull/19378)
+    - Improve compatibility for ALTER TABLE algorithms [#19364](https://github.com/pingcap/tidb/pull/19364)
+    - Fix a schema charset modification panic bug in an uppercase schema. [#19302](https://github.com/pingcap/tidb/pull/19302)
+    - Planner: encode insert/delete/update plan in slow log plan field. [#19269](https://github.com/pingcap/tidb/pull/19269)
+    - Fix inconsistence of Plan between information_schema.statements_summary and explain, which contains tikv/tiflash info. [#19159](https://github.com/pingcap/tidb/pull/19159)
+
++ TiKV
+
+    - Optimize QPS drop when doing UnsafeDestroyRange [#8627](https://github.com/tikv/tikv/pull/8627)
+    - Fix the estimation error for a non-index column with collation enabled. [#8620](https://github.com/tikv/tikv/pull/8620)
+    - Error-code: support to generate metafile [#8619](https://github.com/tikv/tikv/pull/8619)
+    - Add perf statistics for cf scan detail [#8618](https://github.com/tikv/tikv/pull/8618)
+    - Fix the deadlock between the PD client thread and other threads calling PD sync requests. [#8612](https://github.com/tikv/tikv/pull/8612)
+    - Fix a panic issue if a TiKV runs very slow during conf change. [#8497](https://github.com/tikv/tikv/pull/8497)
+    - Add perf context panel for grafana template [#8467](https://github.com/tikv/tikv/pull/8467)
+    - Update jemalloc to 5.2.1 [#8463](https://github.com/tikv/tikv/pull/8463)
+    - Fix the issue that green GC may miss locks during transferring regions. [#8460](https://github.com/tikv/tikv/pull/8460)
+    - Support float CPU quota in cgroup. [#8427](https://github.com/tikv/tikv/pull/8427)
+
++ PD
+
+    - Add `initial-cluster-token` configuration to avoid different clusters communicate during bootstrap [#2922](https://github.com/pingcap/pd/pull/2922)
+    - Fix the unit of store limit rate when the mode is auto [#2826](https://github.com/pingcap/pd/pull/2826)
+    - Fix the issue that some scheduler persist config without solving error [#2818](https://github.com/tikv/pd/pull/2818)
+    - fix empty http response in scheduler [#2871](https://github.com/tikv/pd/pull/2871) [#2874](https://github.com/tikv/pd/pull/2874)
+
++ TiFlash
+
+    - `Do not optimize applying snapshot by multi-thread when raftstore.snap-handle-pool-size is 0` [#1086](https://github.com/pingcap/tics/pull/1086)
+    - `Fix bug causes segment fault while computing table sync status` [#1077](https://github.com/pingcap/tics/pull/1077)
+    - For invalid collation, treated as `utf8mb4_bin` in TiFlash [#1069](https://github.com/pingcap/tics/pull/1069)
+    - Added some Grafana panels about write stall, and added some settings to avoid write stall. [#1051](https://github.com/pingcap/tics/pull/1051)
```

Comment:
```suggestion
    - Added some Grafana panels about write stall, and added some settings to avoid write stall. [#1051](https://github.com/pingcap/tics/pull/1051)

## New Features

+ BR

    - Support compressing backup files using zstd, lz4 or snappy [#404](https://github.com/pingcap/br/pull/404)

+ TiCDC

    - Support output maxwell data format [#869](https://github.com/pingcap/ticdc/pull/869)

## Bug Fixes

+ TiCDC

    - Fix the issue that TiCDC leaks memory in some cases [#942](https://github.com/pingcap/ticdc/pull/942)
    - Fix the issue that TiCDC may panic in Kafka sink [#912](https://github.com/pingcap/ticdc/pull/912)
    - Fix the issue that CRTs may less than resolved ts in puller [#927](https://github.com/pingcap/ticdc/pull/927)
    - Fix the issue that change feed may blocked by MySQL driver [#936](https://github.com/pingcap/ticdc/pull/936)

+ BR

    - Fix a panic during checksum [#479](https://github.com/pingcap/br/pull/479)
    - Fix a panic after PD changes leader [#496](https://github.com/pingcap/br/pull/496)

+ Dumpling

    - Fix the issue that the NULL value for binary type is not handled properly [#137](https://github.com/pingcap/dumpling/pull/137)

+ TiDB-Lightning

    - Fix the issue that write and ingest all retry failed will just treat it as success [#381](https://github.com/pingcap/tidb-lightning/pull/381)
    - Fix the issue that some checkpoint update may not be written to db before exit [#386](https://github.com/pingcap/tidb-lightning/pull/386)

## Improvements

+ TiCDC

    - Skip resolved lock during puller initialization [#910](https://github.com/pingcap/ticdc/pull/910)
    - Reduce PD write frequency [#937](https://github.com/pingcap/ticdc/pull/937)

+ BR

    - Add real time cost in summary log [#486](https://github.com/pingcap/br/issues/486)

+ Dumpling

    - Support output `INSERT` with column names [#135](https://github.com/pingcap/dumpling/pull/135)
    - Unify `--filesize` and `--statement-size` definition with mydumper's [#142](https://github.com/pingcap/dumpling/pull/142)

+ TiDB-Lightning

    - Split and ingest region size more precise [#369](https://github.com/pingcap/tidb-lightning/pull/369)

+ TiDB-Binlog

    - Support go time package format gc time (#996)

```

Reply:
Added.